src/resources/api_nw_newwin.js

```diff
@@ -369,15 +369,23 @@ NWWindow.prototype.toggleKioskMode = function() {
   currentNWWindowInternal.toggleKioskModeInternal(this.cWindow.id);
 };
 
-NWWindow.prototype.showDevTools = function(frm, callback) {
-  var id = '';
-  if (typeof frm === 'string')
-    id = frm;
-  var f = null;
+/**
+ * Open the devtools to inspect the new window.
+ * 
+ * @param {string | HTMLIFrameElement} iframe
+ * @param {Function} callback
```

Comment:
This is probably fine. But ff you wanted to give it a full JSDoc definition, then you would make another doc block above this one, like so:
```js
/**
 * Description of function
 *
 * @callback {Function} NAMEOFFUNCTION
 * @param    {type}     argName         Argument description
 * @return   {type}                     return description
 */
```
Then you'd replace the type here with a reference to it
```suggestion
 * @param {NAMEOFFUNCTION}             callback  Description of argument
```

You could also make the doc block definition in another JS file and then export it from the file and import it here:

```js
/**
 * @type {NAMEOFFUNCTION}
 */
let NAMEOFFUNCTION

module.exports = {
  NAMEOFFUNCTION
};
```
Then `const { NAMEOFFUNCTION } = require('./api-type-definitions.js');` if you were to reuse the same definition in multiple files.

* Example: https://github.com/nwutils/create-desktop-shortcuts/blob/main/api-type-definitions.js

Reply:
Thanks! I'll keep this in mind. My current goal is to add code comments so that other people reading the code know exactly what the function does. I think I'll add that in later. I'm trying to keep this PR as small as possible so it can be reviewed easily by Roger.

cc @rogerwang